TombEngine/Game/control/box.h

```diff
@@ -99,10 +50,10 @@ struct OVERLAP
 	int flags;
 };
 
-struct BiteInfo
+/*struct BiteInfo
```

Comment:
Don't leave commented blocks like that. If this code will be needed in future, add a comment tagged with `TODO` in front. Otherwise, delete it.

Reply:
Done.